scripts/dts/python-devicetree/src/devicetree/edtlib.py

```diff
@@ -642,13 +642,15 @@ class Node:
       node, sorted by index. The list is empty if the node does not have any
       pinctrl-<index> properties.
 
-    bus:
+    buses:
       If the node is a bus node (has a 'bus:' key in its binding), then this
-      attribute holds the bus type, e.g. "i2c" or "spi". If the node is not a
-      bus node, then this attribute is None.
+      attribute holds the list of supported bus types, e.g. ["i2c"], ["spi"]
+      or ["i3c", "i2c"] if multiple protocols are supported via the same bus.
+      If the node is not a bus node, then this attribute is None.
```

Comment:
Making it an empty list in this case would be simpler for the user as shown above.